ouroboros-network/src/Ouroboros/Network/Diffusion/P2P.hs

```diff
@@ -814,12 +824,22 @@ runM Interfaces
 
       churnModeVar <- newTVarIO ChurnModeNormal
 
-      peerSelectionTargetsVar <- newTVarIO $ daPeerSelectionTargets {
+      -- ensure that peer selection governor does not change targets
+      -- in the initial starting phase when churn governor should be
+      -- in control. See comment directly below.
+      churnMutex <- newEmptyTMVarIO
+
+      peerSelectionTargetsVar <- newTVarIO $
+        -- because peer selection governor starts up in TooOld state
```

Comment:
I don't get why this comment is made here

Reply:
Done.